lib/foreman_remote_execution/engine.rb

```diff
@@ -138,6 +139,13 @@ class Engine < ::Rails::Engine
           parent: :monitor_menu,
           after: :audits
 
+        menu :labs_menu, :job_wizard,
+          :url_hash => { :controller => 'job_wizard', :action => :index },
+          caption: N_('Job wizard'),
+          parent: :lab_features_menu,
+          :url => 'experimental/job_wizard',
+          after: :host_wizard
```

Comment:
Please don't mix the hash syntax

Reply:
What should I replace it with? or should I just remove `:url_hash => { :controller => 'job_wizard', :action => :index },`?

Reply:
Either have all the key-value pairs as `{caption: 'caption', parent: :lab_features_menu}` or `:caption => 'caption', :parent => :labe_features menu}`. Either one is fine, as long as it is not a mix of both

Reply:
Oh yeah, that makes sense, changed